packages/forma-36-website/src/pages/guidelines/copy/grammar/word-choice/index.mdx

```diff
@@ -0,0 +1,46 @@
+# Word choice
+
+## Use contractions when possible
+
+To make a sentence more scannable, use contractions when possible. Note that in some cases the words can have more emphasis when they are not contracted (**Can not** has greater emphasis than **Can't**).
+
+![Use contractions when possible](/contractions.jpg)
+
+## Address users based on context (Your, My, We)
+
+Determine the appropriate pronoun based on the context. Focus on the user and the actions the user is taking whenever possible. Avoid mixing prounouns ("my" and "your”) in the same section— It can cause confusion to see both forms of addressing the user in the same context.
+
+### Second person (“you” or “your”)
+
+Use this style for most situations that require a pronoun (when the web app is speaking directly to the user).
```

Comment:
`when the web app is speaking directly to the user` -> `when speaking directly to the user`